aidants_connect_web/models.py

```diff
@@ -561,6 +561,31 @@ def duration_for_humans(self) -> int:
         # and 0 for a computer.
         return duration_for_computer.days + 1
 
+    @property
+    def should_display_revocation_attestation(self) -> bool:
+        return (
+            self.is_revoked
+            and not self.mandat.was_explicitly_revoked
+            and self.was_explicitely_revoked
+        )
+
+    @cached_property
+    def was_explicitely_revoked(self) -> bool:
+        """
+        :return: `True` if the authorization's revocation date is considered
+            different from the mandate's computed revocation date, `False` otherwise.
+            **Notes:** The mandate's computed revocation date is the date of the most
+            recently revoked authorization. The authorization's revocation date will be
+            considered different if it was revoked outside a time window of 30 seconds
+            around the mandate's computed reocation date. I.E., if it was revoked at
+            least 15 seconds before or after the mandate.
+        """
+        return (
+            self.revocation_date < self.mandat.revocation_date - timedelta(seconds=15)
+        ) or (
+            self.revocation_date > self.mandat.revocation_date + timedelta(seconds=15)
+        )
```

Comment:
Du coup pour alléger on peut faire quelque chose comme ceci : 

```suggestion
        return abs(self.revocation_date - self.mandat.revocation_date) > timedelta(seconds=15)
```

Reply:
On peut faire ça ? Comment ça marche ? 😮

Reply:
J'ai voulu essayer, j'ai essayé dans la console python, ça a marché 🤷

Reply:
(Et ça marche bien sur le python 3.7.9 de prod)

Reply:
Je viens de voir [ton explication ](https://github.com/betagouv/Aidants_Connect/pull/350#discussion_r625612607). Y'a pas à dire : Python, c'est bien foutu, quand-même.

Reply:
❤️ 🐍